lib/trample/search.rb

```diff
@@ -166,6 +166,23 @@ def records!
       records
     end
 
+    #This implementation is not using scroll and scan of elastic
+    #https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html
+    #In future versions we hope to replace this with elastic in-built implementation
+    #
+    def find_in_batches(batch_size = 10_000)
```

Comment:
I think this and removing the offset variable if not needed are the only blockers